neo/Wallets/Wallet.cs

```diff
@@ -55,15 +58,78 @@ public virtual void Dispose()
         {
         }
 
+        private List<(UInt160 Account, BigInteger Value)> FindPayingAccounts(List<(UInt160 Account, BigInteger Value)> orderedAccounts, BigInteger amount)
+        {
+            var result = new List<(UInt160 Account, BigInteger Value)>();
+            BigInteger sum_balance = orderedAccounts.Select(p => p.Value).Sum();
+            if (sum_balance == amount)
+            {
+                result.AddRange(orderedAccounts);
+                orderedAccounts.Clear();
+            }
+            else
+            {
+                for (int i = 0; i < orderedAccounts.Count; i++)
+                {
+                    if (orderedAccounts[i].Value < amount)
+                        continue;
+                    if (orderedAccounts[i].Value == amount)
+                    {
+                        result.Add(orderedAccounts[i]);
+                        orderedAccounts.RemoveAt(i);
+                    }
+                    else
+                    {
+                        result.Add((orderedAccounts[i].Account, amount));
+                        orderedAccounts[i] = (orderedAccounts[i].Account, orderedAccounts[i].Value - amount);
+                    }
+                    break;
```

Comment:
What happens if we want to pay 100, and we have 20 and 80? i think that we don't have the result of both address

Reply:
https://github.com/neo-project/neo/blob/e609a767b972d063d3f3cc35c1ae19493de88a09/neo/Wallets/Wallet.cs#L106-L132

It will be handled here.